<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMooreTwoStateSelect</class>
 <widget class="QWidget" name="CMooreTwoStateSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/common/moore1.png);
</string>
   </property>
   <widget class="QLabel" name="mRgMsbNoHint">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>156</y>
      <width>30</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">10</string>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelHint_1">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>156</y>
      <width>46</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelHint_2">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>116</y>
      <width>20</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMs2MsbHint">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>116</y>
      <width>46</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">23</string>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelHint_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>47</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">p0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mLabelHint_5">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>46</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">p1</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMs1MsbHint">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>47</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">p9</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mYDimHint">
    <property name="geometry">
     <rect>
      <x>346</x>
      <y>172</y>
      <width>46</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">23</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRgVal">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>190</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">ABh</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mYVal">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>151</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0xABC</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMs1Val">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>74</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mMs2SVal">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>111</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">ABh</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mMS1P0Val">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>20</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mMS1P1Val">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>20</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mMS1PnVal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mMS2S1Val">
    <property name="geometry">
     <rect>
      <x>127</x>
      <y>36</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">1Ah</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mMS2S0Val">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>36</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">1Ah</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mSVal">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>214</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">ABh</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMs1SelVal">
    <property name="geometry">
     <rect>
      <x>354</x>
      <y>51</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0xAB</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
